locale/zhCN.lua

```diff
@@ -1,26 +1,26 @@
 local L = select(2, ...)('zhCN')
 
 -- Withered scenario, pulled from LibBabble-SubZone-3.0
-L['The Collapse'] = '大陷坑'
```

Comment:
This one needs to be 100% identical to the Suramar withered scenario zone, which I've pulled from LibBabble-SubZone and should be correct already.

Reply:
select zhcn locale client to check last year when translating but ok dont change that until check again

Reply:
If you can, please provide a screenshot to confirm either way what the name actually says.

Reply:
well i cant take screenshot at once...would be later

Reply:
in fact i suggest use mapID/instancetype to disable automatic O.o

Reply:
I was not able to do that before, but I will check again with the new map system, thanks for reminding me.